OracleAccessManagement/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -0,0 +1,275 @@
+#!/bin/bash
+#
```

Comment:
Please remove all but one line between the shebang and the copyright.

Reply:
Will do it.